### PR TITLE
Port to python3

### PR DESCRIPTION
**Pull request by ssjunnebo**

### CODE REVIEW

File: make_dashboards/make_dashboards.py [hunk 1]
```diff
@@ -26,12 +24,11 @@
     p_version = f.read()
 
 # Command line options
-@click.command( context_settings = dict( help_option_names = ['-h', '--help'] ))
+@click.command(context_settings=dict(help_option_names=['-h', '--help']))
 @click.option('--outdir', '-o', required=True, help = "Create dashboards in the specified output directory.")
-@click.option('--demo', is_flag=True)
```
Comment (remiolsen) [inline make_dashboards/make_dashboards.py:31]: Ah there was still some leftovers from the old internal dashboard.

File: requirements.txt [hunk 2]
```diff
@@ -1,30 +1,3 @@
...
-pycryptodome>=3.8.2
-pyparsing>=2.0.1
-python-dateutil>=1.5
-pytz>=2014.7
-requests>=2.22.0
-simplejson>=3.6.4
-six>=1.8.0
-tornado>=4.0.2
-urllib3>=1.25.3
-wsgiref>=0.1.2
```
Comment (remiolsen) [inline requirements.txt:30]: What were we thinking? I also spot some more cruft from ancient times that has been removed :+1:

File: make_dashboards/make_dashboards.py [hunk 3]
```diff
@@ -26,12 +24,11 @@
...
 @click.option('--genstat_url', '-g', default="https://genomics-status.scilifelab.se")
 @click.version_option(p_version)
-def make_dashboards(outdir, demo, genstat_url):
```
Comment (remiolsen) [inline make_dashboards/make_dashboards.py:34]: Leftover from the now removed internal dashboard. 🧹

**Review by remiolsen — APPROVED** [review]
Thanks a lot for doing this. 

We have been using such and old version of jinja2. My only concern is that the external dashboard could be a little bit fragile to get right on our 1920x1080 displays, and even though this is probably only a front-end issue. Do you have an example .html output with these changes?

**Comment by ssjunnebo** [comment]
> Thanks a lot for doing this.
> 
> We have been using such and old version of jinja2. My only concern is that the external dashboard could be a little bit fragile to get right on our 1920x1080 displays, and even though this is probably only a front-end issue. Do you have an example .html output with these changes?

No problem! This should be the last thing we are still running on python2 🎉  I've sent you the files on slack 👍

**Comment by remiolsen** [comment]
Great! It looks normal on the closest screen I could find. I'm happy 🎉

**Pull request merged**